src/controllers/bookController.js

```diff
@@ -0,0 +1,18 @@
+const  bookService  = require("../services/bookService");
+const { catchAsync } = require("../utils/error");
+
+const getAllBooks = catchAsync(async (req, res) => {
+  const books = await bookService.getAllBooks();
+  res.status(200).json(books);
```

Comment:
함수가 종료되는 것을 의미하는 return 키워드를 작성해주세요!
```suggestion
  return res.status(200).json(books);
```

Reply:
수정 완료하였습니다!